go.mod

```diff
@@ -10,3 +10,5 @@ require (
 	github.com/sirupsen/logrus v1.8.0
 	golang.org/x/net v0.0.0-20210405180319-a5a99cb37ef4 // indirect
 )
+
+replace github.com/infobloxopen/infoblox-go-client/v2 => github.com/skudriavtsev/infoblox-go-client/v2 v2.1.2-0.20221107170905-7c162bba78b8
```

Comment:
Why is [this](https://github.com/skudriavtsev/infoblox-go-client/commit/7c162bba78b8) functionality is not merged into inofblox-client repo yet?